Jenkinsfile.continuous

```diff
@@ -0,0 +1,44 @@
+// This pipeline is used to build and publish new versions of the stack when changes are merged into
+// the master branch.
+pipeline {
+    agent { label 'upbound-gce' }
+
+    options {
+        disableConcurrentBuilds()
+        timestamps()
+    }
+
+    environment {
+        DOCKER = credentials('dockerhub-upboundci')
+        CROSSPLANE_CLI_RELEASE = 'v0.2.0'
+    }
+
+    stages {
+        stage('Prepare') {
+            steps {
+                sh 'mkdir bin'
+                sh "curl -sL https://raw.githubusercontent.com/crossplaneio/crossplane-cli/${CROSSPLANE_CLI_RELEASE}/bootstrap.sh | env PREFIX=${WORKSPACE} RELEASE=${CROSSPLANE_CLI_RELEASE} bash"
+            }
+        }
+        stage('Promote Release') {
+
+            steps {
+                // The build step turns this into a "dirty" environment from the perspective of `git describe`,
+                // so we set the version once at the beginning and use it for both the build and publish steps.
+
+                sh """STACK_VERSION=$( git describe --tags --dirty --always )
+                      STACK_VERSION=\${STACK_VERSION} ./bin/kubectl-crossplane-stack-build
```

Comment:
will this exit the pipeline if it fails?

Reply:
Yes, it will. Here's an example: https://jenkinsci.upbound.io/job/crossplane/job/sample-stack-wordpress/job/tag/job/automated-builds/3/console